### ISSUE TITLE
Move RainTPL3 pages to RainTPL4

### ISSUE THREAD

**Issue by webnull** [Panthera-Framework/raintpl4#8]
Hello,
Please move all old RainTPL3 test pages (example-*.php) to RainTPL4

Thanks,
Damian

**Comment by webnull** [comment]
@lnxmen, could you do something here? :)

**Comment by Abhinav1217** [comment]
Someone should move documentations also.  There are several difference and new features from RainTPL3 and this one.  They need to be documented too and with example.

**Comment by webnull** [comment]
I agree. We need a documentation. Who want to take this? :D

2015-11-23 14:07 GMT+01:00 Abhinav Kulshreshtha <notifications@github.com>:

**Comment by Abhinav1217** [comment]
I would love to help, But I am yet to familiarize myself with new features of RT4.  Would you be ok with first PR documenting what it inherit with RT3, and then multiple updating PR as I learn more about it.

**Comment by webnull** [comment]
Could you check what is backwards compatible with RainTPL3 and make some documentation pages? :)

**Comment by Abhinav1217** [comment]
Sure.  I can do that. Give me a few days.  I had already started documenting  RT4 last time I messaged here. Got busy in other stuffs as days passed. :)

**Comment by webnull** [comment]
Yes sure. Please note that in next 2 months I will be heavy busy as I my son will born in a few days.
If you will have any quick questions I could answer you via Skype. Also Matthew (lnxmen) is a person you could try to contact in case of questions, but mainly I implemented all features.

**Comment by Abhinav1217** [comment]
Ok. Good for me, I can take time and do it right. I will ping you on skype if needed.

Congratulations on the birth of your son.

**Comment by webnull** [comment]
Thank you. And thank you for participating in the project. I hope more people will join soon to work on open source projects I started.

I have more projects in my "portfolio" I'm just releasing:
- Panthera Framework 2 (PHP framework)
- Panthera Desktop Framework (Python framework, provides basic kernel + logging, configuration management, easier shell arguments parser builtin than raw getopt/argsparse, database support, Qt4/Qt5 gui rendering)
- RaspAP (web panel written in PHP and Python that easily allows to make a router from Linux machine, eg. laptop or Raspberry Pi or any other machine)
- Subget (downloads subtitles for movies from multiple sources, works on Linux, FreeBSD and Windows - this last is very poorly supported, this project needs a strong refactoring, it's old, but still gets more popular on github and works...)
- And of course RainTPL4

I invite you to participate in any project you like and use. We look for contributors hehe :smile:
